export const registerUser = async (req, res, next) => {
    try {
        res.send("a")
    } catch (error) {
        next(error)
    }
}

export const loginUser = async (req, res, next) => {
    try {
        res.send("a")
    } catch (error) {
        next(error)
    }
}